### DISCUSSION TITLE
localsend run chrome faild

### DISCUSSION THREAD

**Discussion by xiaoxiaolexlh** [localsend/localsend#2026] [Discussion (English, International)]
flutter run -d chrome failed

a lot error, like this

AppData/Local/Pub/Cache/hosted/pub.dev/win32-5.8.0/lib/src/combase.dart:22:26: Error: Only  JS interop members may be 'external'. Try removing the 'external' keyword or adding a JS interop annotation.   external VTablePointer lpVtbl;

**Comment by Tienisto** [comment]
Web is currently not supported. See #9
